test/modules/TestResultCoordinator/Topology.cs

```diff
@@ -1,13 +1,13 @@
-// Copyright (c) Microsoft. All rights reserved.
-namespace TestResultCoordinator
-{
-    public enum Topology
-    {
-        // A single edge device.
-        SingleNode,
-
-        // Multiple edge devices connected together in some toplogy.
-        // Runs an extended test suite relative to `SingleNode`.
-        Nested
-    }
-}
```

Comment:
What was the change to this file? We may want to revert the change.

Reply:
When I originally pulled in these, it had an extra newline at the bottom of the file initially